test/ILLink.RoslynAnalyzer.Tests/RequiresAssemblyFilesAnalyzerTests.cs

```diff
@@ -34,6 +48,30 @@ public sealed class RequiresAssemblyFilesAttribute : Attribute
 				expected);
 		}
 
+		static Task VerifyRequiresAssemblyFilesCodeFix (
+			string source,
+			string fixedSource,
+			DiagnosticResult[] baselineExpected,
+			DiagnosticResult[] fixedExpected,
+			int? numberOfIterations = null)
+		{
+			var test = new VerifyCS.Test {
+				TestCode = source + rafDef,
+				FixedCode = fixedSource + rafDef,
+			};
+			test.ExpectedDiagnostics.AddRange (baselineExpected);
+			test.TestState.AnalyzerConfigFiles.Add (
+						("/.editorconfig", SourceText.From (@$"
+is_global = true
+build_property.{MSBuildPropertyOptionNames.EnableSingleFileAnalyzer} = true")));
```

Comment:
`EnableSingleFileAnalyzer`? These tests probably shouldn't be passing 😄

Reply:
I agree that shouldnt be passing because we are missing setting IncludeAllContentForSelfExtract to true. Otherwise, we set this MSBuild property and check for it in [RequiresAssemblyFilesAnalyzer.cs](https://github.com/mono/linker/blob/main/src/ILLink.RoslynAnalyzer/RequiresAssemblyFilesAnalyzer.cs#L67)

Reply:
Ah, my mistake, I thought we changed the name -- but it's just the attribute.